packages/api/src/utils/billing-types.ts

```diff
@@ -0,0 +1,59 @@
+export type StripePaymentMethodId = string;
```

Comment:
no strong feelings against this, but we are using jsdocs for types everywhere else in this repo. Would be great to uniform how we want to proceed with the whole team. I am up to go with types as in your PR, but prefer us to be consistent rather than having different styles all around.

Could you raise this with the team? (not blocking the PR in any way, we can revert back if we need to merge this ASAP)

Reply:
> Could you raise this with the team? (not blocking the PR in any way, we can revert back if we need to merge this ASAP)

Def 👍 

> Would be great to uniform how we want to proceed with the whole team.

Personally I think it's not that important to be fully uniform about this. The jsdocs `@typedefs` are useful to create type aliases in the js files for intra-file use (it cuts down on many big `import(...)` in the jsdocs). But the jsdoc syntax is kludgy and not always able to express the whole type system, so then it makes sense to use .ts files. Even though there are ts files for some types, because those ts files are only linked to from jsdocs, all the js should run without transpilation still.

I don't mind at all establishing some kind of more strict norm, just presenting the case for incrementalism/agility over uniformity.

Reply:
https://github.com/web3-storage/web3.storage/discussions/1858